homeassistant/components/config/device_registry.py

```diff
@@ -11,17 +13,27 @@
     vol.Required('type'): WS_TYPE_LIST,
 })
 
+WS_TYPE_UPDATE = 'config/device_registry/update'
+SCHEMA_WS_UPDATE = websocket_api.BASE_COMMAND_MESSAGE_SCHEMA.extend({
+    vol.Required('type'): WS_TYPE_UPDATE,
+    vol.Required('device_id'): str,
+    vol.Required('area_id'): str,
```

Comment:
should this be required? are devices required to be attached to an area? If this is extended for general device updates in the future and devices aren't required to be attached to an area this may be an issue.

Reply:
This is in the device registry. The method is to update the device, and it only supports updating area right now. So yeah. But I will make it optional.